packages/config/config/devices/0x027a/zen26.json

```diff
@@ -314,5 +314,11 @@
 				}
 			]
 		}
-	]
+	],
+	"metadata": {
+		"inclusion": "1. Initiate inclusion (pairing) in the app (or web interface).\n2. Finalize inclusion at the switch. Tap the upper paddle of the dimmer 3 times quickly. The LED indicators will blink to signal communication and remain on for 2 seconds to confirm inclusion",
+		"exclusion": "1. Bring your Z-Wave gateway (hub) close to the switch if possible\n2. Put the Z-Wave hub into exclusion mode \n3. Tap the lower paddle on the dimmer 3 times quickly\n4. Your hub will confirm exclusion and the device will disappear from your\ncontroller's device list",
```

Comment:
Uhhhh. Even if they have this in their manual, I don't think we should ever advertise moving the controller.
https://github.com/zwave-js/node-zwave-js/issues/1995

Reply:
Agreed :D